packages/cf-style-container/README.md

```diff
@@ -2,10 +2,105 @@
 
 > Cloudflare Style Container
 
-Set of high order components for fela based applications.
+Set of high order components and other helpers for fela based applications.
 
 ## Installation
 
 ```sh
 $ npm install cf-style-container
 ```
+
+### Aliased functions from fela and react-fela
+
+We proxy/alias some useful functions from fela without changing their behaviour. See the original documentation for more details. We wrap all Fela APIs so we can eventually switch Fela to a different CSS in JS lib if ever needed.
```

Comment:
Polished is just a bunch of "css preprocessor mixins" for CSS-in-JS solutions, basically, just a bunch of simple, basically pure functions. Fela has a lot of behavior that can't be expressed by these simple signatures. Replacing things is never really easy. Back in the days, people thought having an ORM layer would make switch the RDBMS easy, I lol'ed. Anyway, I digress, it's not important.